src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-details/iscsi-target-details.component.html

```diff
@@ -12,10 +12,9 @@
         <i [class]="node.data.cdIcon"></i>
         <span>{{ node.data.name }}</span>
         &nbsp;
-        <span class="badge"
-              [ngClass]="{'badge-success': ['logged_in'].includes(node.data.status), 'badge-danger': ['logged_out'].includes(node.data.status)}">
+        <cds-tag [class]="{'tags-success': ['logged_in'].includes(node.data.status), 'tags-danger': ['logged_out'].includes(node.data.status)}">
```

Comment:
why not `ngClass` ?  It is optimized for Angular's change detection and works efficiently in scenarios where classes are toggled frequently.